platform/darwin/docs/theme/templates/header.mustache

```diff
@@ -1,28 +1,33 @@
-<header class="header">
-  <p class="header-col header-col--primary">
-    <a class="header-link" href="{{path_to_root}}index.html">
-      <img class="header-icon" src="{{path_to_root}}img/mapbox.svg" alt="{{module_name}} Docs"/>
-    </a>
-    {{#doc_coverage}} ({{doc_coverage}}% documented){{/doc_coverage}}
-  </p>
-
-  {{#github_url}}
-    <p class="header-col header-col--secondary">
-      <a class="header-link" href="{{github_url}}">
-        <svg class="header-icon" viewBox="0 0 33 33" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-		      <path stroke="none" stroke-width="1" fill="white" fill-rule="evenodd" d="M16.6,0.4 C7.6,0.4 0.3,7.7 0.3,16.7 C0.3,23.9 5,30 11.4,32.2 C12.2,32.3 12.5,31.8 12.5,31.4 L12.5,28.6 C8,29.6 7,26.4 7,26.4 C6.3,24.5 5.2,24 5.2,24 C3.7,23 5.3,23 5.3,23 C6.9,23.1 7.8,24.7 7.8,24.7 C9.3,27.2 11.6,26.5 12.5,26.1 C12.6,25 13.1,24.3 13.5,23.9 C9.9,23.5 6.1,22.1 6.1,15.8 C6.1,14 6.7,12.6 7.8,11.4 C7.6,11 7.1,9.3 8,7.1 C8,7.1 9.4,6.7 12.5,8.8 C13.8,8.4 15.2,8.3 16.6,8.3 C18,8.3 19.4,8.5 20.7,8.8 C23.8,6.7 25.2,7.1 25.2,7.1 C26.1,9.3 25.5,11 25.4,11.4 C26.4,12.5 27.1,14 27.1,15.8 C27.1,22.1 23.3,23.4 19.7,23.8 C20.3,24.3 20.8,25.3 20.8,26.8 L20.8,31.3 C20.8,31.7 21.1,32.2 21.9,32.1 C28.4,29.9 33,23.8 33,16.6 C32.9,7.7 25.6,0.4 16.6,0.4 L16.6,0.4 Z"></path>
-        </svg>
-        View on GitHub
+<header class="header-container">
+  <div class="limiter">
+    <div class="header">
+    <p class="header-col header-col--primary">
+      <a class="header-link" href="{{path_to_root}}index.html">
+        <img style="height: 25px;"  class="header-icon" src="{{path_to_root}}img/mapbox.svg" alt="{{module_name}} Docs"/>
+        <span class='header-tag'>iOS SDK Reference</span>
```

Comment:
You do have to append “Docs” or “Reference” after `{{module_name}}`, because the script looks specifically for the phrases “Mapbox Docs” and “Mapbox Reference”.